ansible/roles/dev-desktop/tasks/main.yml

```diff
@@ -0,0 +1,88 @@
+---
```

Comment:
Can we make the default shell `bash` instead of `sh`?

Reply:
My default shell after login is

Reply:
Ah interesting, only the admin users have this, everyone else has `sh` 😕 looking into it

Reply:
One problem: all existing users now are stuck with `sh` 😆 I'll patch them manually, but in the future all deploys will use bash

Reply:
Done, you may need to delete the `.vscode-server` folder on the server as it remembers the shell at setup time out of some reason

Reply:
We'll definitely recreate the base machine (not just Ansible deploy) before the next stage, so that's a temporary condition.